samples/net/wpanusb/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _wpanusb-sample:
+
+WPANUSB sample
+##############
+
+Overview
+********
+
+This application exports ieee802154 radio over USB to be used in other
+Operation Systems like Linux. In this scenario Linux SoftMAC driver would
```

Comment:
change to:

    This application exports ieee802154 radio over USB to be used in
    other OSes such as Linux.  For Linux, the ieee802154 stack would be
    implemented using the Linux SoftMAC driver.